testsuite/MDAnalysisTests/analysis/test_dihedrals.py

```diff
@@ -71,7 +109,61 @@ def test_protein_ends(self, universe):
         with pytest.warns(UserWarning):
             rama = Ramachandran(universe.select_atoms("protein")).run()
 
+    def test_None_removal(self):
+        with pytest.warns(UserWarning):
+            u = mda.coordinates.MMTF.fetch_mmtf('19hc')
```

Comment:
What happens if `fetch_mmtf()` cannot download the PDB? You should either XFAIL the test if this happens or include a gzipped 19hc.pdb.gz as a data file.